e2e/__tests__/jasmineAsync.test.js

```diff
@@ -125,6 +125,15 @@ describe('async jasmine', () => {
     expect(json.testResults[0].message).toMatch(/concurrent test fails/);
   });
 
+  it("doesn't execute more than 5 tests simultaneously", () => {
```

Comment:
should this test not live in `jasmineAsync` since it works for circus as well? No biggie, I guess

Reply:
I checked the tests for circus but `concurrent` didn't seem to be tested 😕

Reply:
The e2e tests are ran with both jasmine and circus (unless they use `skipOnJasmine`)